src/plugins/data/public/ui/search_bar/search_bar.tsx

```diff
@@ -68,6 +68,12 @@ export interface SearchBarOwnProps {
 
   onRefresh?: (payload: { dateRange: TimeRange }) => void;
   indicateNoData?: boolean;
+
+  placeholder?: string;
+  isClearable?: boolean;
+  iconType?: string;
```

Comment:
What's the `iconType`? It doesn't seem to make any effect? 

----

Probably because `search` is hardcoded by mistake: https://github.com/elastic/kibana/blob/eea6f82e4e436abdf28266127a801a57e3c2dcad/src/plugins/data/public/ui/query_string_input/query_string_input.tsx#L701
could you please adjust it also?

Reply:
done in [`e8ec7a1` (#91814)](https://github.com/elastic/kibana/pull/91814/commits/e8ec7a11a7034af5bd939913088fe2aacc874e7c)